packages/stacked_services/lib/src/navigation/navigation_service.dart

```diff
@@ -187,10 +187,11 @@ class NavigationService {
     int? id,
     bool preventDuplicates = true,
     Map<String, String>? parameters,
+    RouteTransitionsBuilder? transition,
```

Comment:
I think this is a fix for adding transtion to navigation with named route right?

Reply:
Yup! 👍